src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -247,8 +247,10 @@ trait WorkflowComponent {
     }
 
     def loadFailedInputResolutions(failureId: Long): ReadAction[Seq[SubmissionValidationValue]] = {
-      val failedInputResolutionsAttributeJoin = findInputResolutionsByFailureId(failureId) join submissionAttributeQuery on (_.id === _.ownerId)
-      unmarshalInputResolutions(failedInputResolutionsAttributeJoin)
+      val failedInputResolutionsAttributeJoin = findInputResolutionsByFailureId(failureId) joinLeft submissionAttributeQuery on (_.id === _.ownerId)
```

Comment:
Your explanation is exactly right, though the terminology you used was slightly off ... 

Previously, we were doing a left join, which returns the intersection of both tables where the two columns in the join match.

In this PR, we are doing a left **outer** join, which returns all rows from the left table, plus either nulls or the matching row from the right table. This is exactly what we need - in this use case, we don't want the left table filtered based on the presence or absence of a row in the right table.